x/aswap/model.go

```diff
@@ -0,0 +1,102 @@
+package aswap
+
+import (
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/migration"
+	"github.com/iov-one/weave/orm"
+)
+
+func init() {
+	migration.MustRegister(1, &Swap{}, migration.NoModification)
+}
+
+const (
+	// BucketName is where we store the swaps
+	BucketName = "aswp"
```

Comment:
Is this a typo or `a` is missing in purpose? :thinking: 

Bucket is representing a single entity. Would it be better to name it after the model, not the module? For example `swap`.

Reply:
It is missing on purpose, I was thinking about swap too and guess that's a good idea.